lunabot_control/scripts/manual_controller.py

```diff
@@ -93,6 +98,7 @@ def __init__(self):
 
         self.driving_mode = "Forwards"
         
+        self._max_speed = rospy.get_param("~max_speed", 2.0) * 60 / 2 / np.pi # In rad/s converted to RPM
         self.drive_speed_modifier = 1
         self.slow_drive_speed = 0.5
         self.fast_drive_speed = 1
```

Comment:
Github being bad. Deposition speed is wrong two lines down (still at 127)

Reply:
Updated in fix threshold commit